packages/driver/src/cy/commands/actions/select.ts

```diff
@@ -11,6 +11,10 @@ const newLineRe = /\n/g
 export default (Commands, Cypress, cy) => {
   Commands.addAll({ prevSubject: 'element' }, {
     select (subject, valueOrTextOrIndex, options = {}) {
+      if (!valueOrTextOrIndex) {
```

Comment:
Instead of just checking for existence, it would be better to validate that it's the right type. That's what we generally do with other command options. With this implementation, a valid option like `0` will incorrectly trigger the error while an invalid option like `true` will not.

Could you update it to ensure that it's a string or number, update the error message to make it clear those are the only types accepted, and then add a few tests for various valid/invalid values like `0`, `null`, `true`, `false`, `''` (empty string)?

Reply:
Nice catch Chris.